docs/sdks/kotlin-sdk/integrate-with-beyondidentity.md

```diff
@@ -14,7 +17,11 @@ Before calling [`EmbeddedSdk.authenticate()`](overview#authentication), we must
 
 ## Authorize With Beyond Identity
```

Comment:
@ismailmustafa, what do you think? I currently have it as separate sections for each invocation type, but since Step 1 is the same, I think it might be cleaner to keep it as is and break Step 2 - 4 by Invocation Type instead.

Reply:
I think breaking Step 2-4 by Invocation Type might be better. Since we're showing an image for the invocation type, it might be nice to show one for the `Embedded` Authenticator Config too.

Reply:
What would the rough outline look like? Also, I think manual should come before automatic.

Reply:
It would be the same as what is currently live, but Steps 2-4 would have sub-sections for manual and automatic invocation. I can push it in a moment once I swap automatic and manual.

Reply:
I think it might be easier to read if the outline was something like this:

1. Embedded Authenticator Config w /image
2. Manual Invocation w/ image
    - Beyond Identity Authorize URL
    - Invoke URL
    - Redirect URL
3. Automatic Invocation w/image
    - Beyond Identity Authorize URL
    - Invoke URL
    - Redirect URL

Reply:
This looks good to me, but I think we should have Step 2 explain the Invocation Type (like we do in Step 1 for the authenticator config) and move those new images in that step. 

Then Beyond Identity Authorize URL would be Step 3 and all other numbers would move up one etc...